_episodes/00-hpc-intro.md

```diff
@@ -9,109 +9,120 @@ objectives:
 - "Be able to describe what an HPC system is"
 - "Identify how an HPC system could benefit you."
 keypoints:
-- "High Performance Computing (HPC) typically involves connecting to very large computing systems
-  elsewhere in the world."
-- "These HPC systems can be used to do work that would either be impossible or much slower or
-  smaller systems."
-- "The standard method of interacting with such systems is via a command line interface such as
-  Bash."
+- "High Performance Computing (HPC) typically involves connecting to very large computing systems elsewhere in the world."
```

Comment:
I think you missed these

Reply:
Ah, thanks! IIRC, wrapping these used to break things, but that is not the case. Patched in 1f06058